### ISSUE TITLE
mpty Samples of News Content in Train/Test Sets – Feedback & Query

### ISSUE THREAD

**Issue by hsrr** [mazihan880/An-Attributing-Multi-modal-Fake-News-Dataset#11]
Hello, I noticed that there are more than 200 sample news texts in the training set that are empty, and there are about 80 empty ones in the test set. Could you please explain how the specific attribution category judgments are made for these?

**Comment by uuuaaaaa** [comment]
same question

**Comment by mazihan880** [comment]
We have deleted some data due to privacy and political reasons. If you need it, you can filter out these empty data
